lib/msal-core/src/ClientInfo.ts

```diff
@@ -30,6 +31,12 @@ export class ClientInfo {
         this._utid = utid;
     }
 
+    static createClientInfoFromIdToken(idToken:IdToken): string {
+        const clientInfo = {uid: idToken.subject, utid: ""};
```

Comment:
Can you put each property on a new line?

```suggestion
        const clientInfo = {
            uid: idToken.subject, 
            utid: ""
        };
```